src/test/java/net/snowflake/client/jdbc/OpenGroupCLIFuncIT.java

```diff
@@ -10,17 +10,18 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
-import net.snowflake.client.AbstractDriverIT;
 import net.snowflake.client.TestUtil;
 import net.snowflake.client.category.TestCategoryOthers;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 
 /** Test OpenGroup CLI */
 @Category(TestCategoryOthers.class)
-public class OpenGroupCLIFuncIT extends BaseJDBCTest {
-  public static Connection getConnection() throws SQLException {
-    Connection connection = AbstractDriverIT.getConnection();
+public class OpenGroupCLIFuncIT extends BaseJDBCWithSharedConnectionIT {
+
+  @Before
+  public void setSessionTimezone() throws SQLException {
```

Comment:
Could we execute it once, in @BeforeClass instead of setting it for every test?

Reply:
This change has been made.